site/content/docs/user/ingress.md

```diff
@@ -85,35 +85,9 @@ Additional information about Contour can be found at: [projectcontour.io](https:
 
 ### Ingress Kong
 
-Deploy [Kong Ingress Controller (KIC)](https://docs.konghq.com/kubernetes-ingress-controller/2.1.x/concepts/design/).
+Deploy [Kong Ingress Controller (KIC)](https://docs.konghq.com/kubernetes-ingress-controller/latest/).
 
-{{< codeFromInline lang="bash" >}}
-kubectl apply -f https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/single/all-in-one-dbless.yaml
-{{< /codeFromInline >}}
-
-Apply kind specific patches to forward the `hostPorts` to the ingress controller, set taint tolerations, and schedule it to the custom labeled node.
-
-```json
-{{% readFile "static/examples/ingress/kong/deployment.patch.json" %}}
-```
-
-Apply it by running:
-
-{{< codeFromInline lang="bash" >}}
-kubectl patch deployment -n kong proxy-kong -p '{{< minify file="static/examples/ingress/kong/deployment.patch.json" >}}'
-{{< /codeFromInline >}}
-
-Apply kind specific patch to change service type to `NodePort`:
-
-```json
-{{% readFile "static/examples/ingress/kong/service.patch.json" %}}
-```
-
-Apply it by running:
-
-{{< codeFromInline lang="bash" >}}
-kubectl patch service -n kong kong-proxy -p '{{< minify file="static/examples/ingress/kong/service.patch.json" >}}'
-{{< /codeFromInline >}}
+For deployment related information you can visit [Deployment Methods](https://docs.konghq.com/kubernetes-ingress-controller/latest/#deployment-methods)
```

Comment:
Do these docs cover installing with kind anywhere? Or do they work with kind anyhow?

Otherwise we should just drop kong entirely.

Reply:
@BenTheElder this doc donot have any kind specific installation.
OK then i will remove complete kong section